DEPLOYNOTES.md

```diff
@@ -1,5 +1,9 @@
 # Deploy Notes
 
+## 4.11
+
+-   Must configure **METADATA_REPO_REMOTE_URL** and **METADATA_REPO_LOCAL_PATH** in local settings. These will determine locations for (respectively) the remote metadata repository (e.g. "https://github.com/Princeton-CDH/test-geniza-metadata") and the local files for the metadata exposts and syncing (e.g. "~/github/test-geniza-metadata").
```

Comment:
For the annotation exports we have **ANNOTATION_BACKUP_GITREPO** and **ANNOTATION_BACKUP_PATH** — could we make the names more consistent, since they are parallel?

It looks like I didn't do this for the annotation backup script, but would you add a draft note here of the cron job that should be set up to export/sync metadata, and what options it should be run with? (We'll configure the cron job via ansible, but it's nice to have documentation about when it was added and how it should be run.)